src/Delegates/HandlesIndex.php

```diff
@@ -10,16 +10,11 @@
 
 trait HandlesIndex
 {
-    public function getAllIndexes(IndexesQuery $options = null): IndexesResults
+    public function getIndexes(IndexesQuery $options = null): IndexesResults
     {
         return $this->index->all($options ?? null);
     }
 
-    public function getAllRawIndexes(IndexesQuery $options = null): array
-    {
-        return $this->index->allRaw($options ?? []);
-    }
-
```

Comment:
I'm not sure we should remove this method even if alright the user may not need it. But if we do, we should remove `allRaw` method too no? Or it is used somewhere else?

Reply:
Yeah, that's taking my sleep away... I'm not comfortable changing most of the "raw" methods, because, internally the Indexes class uses a "very strange pattern to me".

Reply:
Me too, I removed some of them in some SDK, but I wasn't sure either ... Maybe we should have a discussion about it with @meilisearch/integration-team? WDYT?

Reply:
Yes, let's do it!